go/webhook.go

```diff
@@ -39,7 +39,28 @@ func NewWebhook(secret string) (*Webhook, error) {
 	}, nil
 }
 
+// Verify validates the payload against the svix signature headers
+// using the webhooks signing secret.
+//
+// Returns an error if the body or headers are missing/unreadable
+// or if the signature doesn't match.
 func (wh *Webhook) Verify(payload []byte, headers http.Header) error {
+	return wh.verify(payload, headers, true)
+}
+
+// VerifyIgnoringTolerance validates the payload against the svix signature headers
+// using the webhooks signing secret.
+//
+// Returns an error if the body or headers are missing/unreadable
+// or if the signature doesn't match.
+//
+// WARNING: This function does not check the signature's timestamp.
+// We recommend using the `Verify` function instead.
+func (wh *Webhook) VerifyIgnoringTolerance(payload []byte, headers http.Header) error {
```

Comment:
```suggestion
func (wh *Webhook) VerifyIgnoringTimestamp(payload []byte, headers http.Header) error {
```

I think this better explains what's up.